packages/admin-ui/src/components/LinkDocs.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import newTabProps from "utils/newTabProps";
+import { FaReadme } from "react-icons/fa";
+
+export default function LinkDocs({
+  urlDocs,
+  text
+}: {
+  urlDocs: string;
+  text?: string;
+}) {
```

Comment:
You can simplify the API by renaming `urlDocs` to `href` and rendering children instead of text

```ts
<a style={{ display: "inLine" }} href={urlDocs} {...newTabProps}>
      <FaReadme className="links-icon" />
      {children}
    </a>
```